dusty/systems/nfs/client.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import absolute_import
+
+import logging
+from subprocess import CalledProcessError
+import time
+
+from ...log import log_to_client
+from ...subprocess import call_demoted, check_call_demoted, check_output_demoted
+from ...compiler.spec_assembler import get_all_repos
+
+def mount_active_repos():
+    remount_repos(get_all_repos(active_only=True, include_specs_repo=False))
+
+def remount_repos(repos):
+    _start_nfs_client()
+    for i, repo in enumerate(repos):
+        wait_for_server = not i
```

Comment:
Could we do something like this instead?

`_mount_repo(repo, wait_for_server=True if i==0 else False)`